frontend/src/e2e-test/pages/employee/holiday-periods.ts

```diff
@@ -19,9 +19,10 @@ export class HolidayPeriodsPage {
   }
 
   async clickAddButton() {
-    return this.page.findByDataQa('add-button').click()
+    return this.page.findByDataQa('add-holiday-period-button').click()
   }
 
+  // TODO move questionnaire fields
```

Comment:
Should we create a ticket for this or just leave the TODO here for poor future souls? 😆